src/bosh-director/lib/bosh/director/task_db_writer.rb

```diff
@@ -0,0 +1,15 @@
+module Bosh::Director
+  class TaskDBWriter
+    def initialize(column, task)
+      @column_name = column
+      @task = task
+      @transactor = Transactor.new
+    end
+
+    def write(text)
+      @transactor.retryable_transaction(Bosh::Director::Config.db) do
+        @task.update({@column_name => "#{@task[@column_name]}#{text}"})
```

Comment:
let's avoid pulling down existing value of the column when updating it. Instead we should be able to use sequel [concat](http://stackoverflow.com/questions/14263069/how-to-concat-a-string-with-a-table-field-with-sequel))